testing/trino-testing/src/main/java/io/trino/testing/AbstractTestEngineOnlyQueries.java

```diff
@@ -6049,6 +6049,40 @@ public void testExplainExecuteWithUsing()
         assertEquals(getOnlyElement(result.getOnlyColumnAsSet()), getExplainPlan("SELECT * FROM orders WHERE orderkey < 7", LOGICAL));
     }
 
+    @Test
```

Comment:
Pleas make sure we have tests with:
- ValuesNode without output symbols
- ValuesNode with 0 rows
- both